config/charts/inferencepool/templates/NOTES.txt

```diff
@@ -0,0 +1 @@
+Gateway api inference extension deployed.
```

Comment:
```suggestion
InferencePool deployed.
```

Reply:
Is this file needed at all?

Reply:
Not really no, but we can use it to add more useful information in the future. Whatever we put here will print out after someone runs `helm upgrade` or `helm install`.